content/v2.0/query-data/flux/increase.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Calculate the increase
+seotitle: Calculate the increase in Flux
+list_title: Increase
+description: >
+  Use the [`increase()` function](/v2.0/reference/flux/stdlib/built-in/transformations/aggregates/increase/)
+  to calculate the increase in values since the beginning of the series or table.
+  This function is especially useful when tracking changes in counter values which
+  wrap over time or periodically reset.
+weight: 210
+menu:
+  v2_0:
+    parent: Query with Flux
+    name: Increase
+v2.0/tags: [query, increase, counters]
+list_code_example: |
+  ```js
+  data
+    |> increase()
+  ```
+---
+
+Use the [`increase()` function](/v2.0/reference/flux/stdlib/built-in/transformations/aggregates/increase/)
+to calculate the increase of values in a table.
```

Comment:
Increase of values in a column? (...or track increases across multiple columns in a table?)